### PR TITLE
[WIP] Spatially enabled Solr image

### PR DESCRIPTION
**Pull request by amercader**
* [Solr documentation](https://solr.apache.org/guide/8_11/spatial-search.html)
* Current (to be replaced) [spatial search documentation](http://docs.ckan.org/projects/ckanext-spatial/en/latest/spatial-search.html#choosing-a-backend-for-the-spatial-search) for ckanext-spatial

### CODE REVIEW

**Comment by mtearle** [comment]
The Bioplatforms Australia Data Portal [](https://data.bioplatforms.com/) uses Solr with the JTS library to enable spatial searches of our datasets.    Currently in the process of the upgrade to CKAN 2.9.5 (and associated components).   Happy to volunteer to test/collaborate so I can change https://github.com/BioplatformsAustralia/docker-solr-jts to following a tested/common/released upstream.

File: solr-8/Dockerfile.spatial [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# Add the spatial field type definitions and fields
+
+## RPT
+ENV SOLR_RPT_FIELD_DEFINITION '<fieldType name="location_rpt"   class="solr.SpatialRecursivePrefixTreeFieldType" \
+    spatialContextFactory="JTS"     \
+    autoIndex="true"                \
+    validationRule="repairBuffer0"  \
+    distErrPct="0.025"              \
+    maxDistErr="0.001"              \
+    distanceUnits="kilometers" />'
```
Comment (mtearle) [inline solr-8/Dockerfile.spatial:38]: Compared this with BPA's config in https://github.com/BioplatformsAustralia/dockercompose-bpa-ckan/blob/1dd1f811c269b2be17bacf2072dd55d43ea6cc76/solr/ckan/conf/schema.xml#L84.   This appears based on the instructions for solr-spatial-field in https://github.com/ckan/ckanext-spatial/blob/master/doc/spatial-search.rst

Querying the change of units from degrees to kilometers?

**Comment by amercader** [comment]
@mtearle I used kilometers as that was what the Solr docs suggested but happy to learn more about it
I just released and pushed the spatial enabled images so let me know if they work for you

**Comment by mtearle** [comment]
@amercader  Thanks, I'll do some more digging and raise an issue with thoughts/notes

**Pull request merged**